src/main/java/com/swiftype/appsearch/ClientException.java

```diff
@@ -4,7 +4,6 @@
  * Base exception class for request errors thrown from the Swiftype App Search Client.
  */
 public class ClientException extends Exception {
-  private static final long serialVersionUID = -3184706760786998150L;
```

Comment:
This should stay, since `Exception` is `Serializable`

Reply:
More details https://docs.oracle.com/javase/7/docs/api/java/io/Serializable.html (see the paragraph at the bottom)

Reply:
thanks for the link. I can't code java to get out of a paper bag.